src/server/scripts/Outland/BlackTemple/boss_mother_shahraz.cpp

```diff
@@ -196,15 +196,25 @@ class spell_mother_shahraz_saber_lash_aura : public AuraScript
     }
 };
 
-const Position validTeleportStairsPos[4] =
+const Position validTeleportStairsPos[9] =
 {
+    //Platform teleports
+
+    {945.00f, 149.17f, 197.07483},
+    {956.92f, 153.20f, 197.07483},
+    {933.69f, 154.15f, 197.07483},
+
+    //Floor teleports
+
     {966.87f, 184.45f, 192.84f},
     {927.22f, 187.04f, 192.84f},
     {922.54f, 110.09f, 192.84f},
-    {958.01f, 110.47f, 192.84f}
+    {958.01f, 110.47f, 192.84f},
+    {939.95f, 108.29f, 192.84f},
+    {945.68f, 205.74f, 192.84f}
 };
 
-constexpr float minTeleportDist = 30.f;
+constexpr float minTeleportDist = 35.f;
```

Comment:
This is wrong. Comparing positions of one teleport with a sniff yields the following:
`The distance is: (3D) 30.496243 (2D) 30.202114 (Exact 3D) 32.996243 (Exact 2D) 32.702114 yards.`

Reply:
If exact 2D/3D are around 32 yards (almost 33) should I modify to 32f?
Or Should I return to 30f?

Reply:
I would say return it to 30f.

Reply:
Roger.